.github/ISSUE_TEMPLATE/improve-existing-docs.yaml

```diff
@@ -0,0 +1,47 @@
+name: Improve existing content
+description: Make a suggestion to improve the content in an existing article
+title: "['Improve existing content']: "
```

Comment:
This looks like we'll be encouraging people to have titles like `['Improve existing content']: What I really want the title to be`. Personally, I'd prefer the title to be `What I really want the title to be` because it makes issues more searchable. For example, if we have, over time, hundreds of issues with "improve existing content" in the title, then we won't be able to meaningfully search for anything using the words "improve", "exist", or "content".